torch/csrc/distributed/c10d/Work.hpp

```diff
@@ -34,6 +34,13 @@ enum class OpType : std::uint8_t {
   UNKNOWN = 100,
 };
 
+// TODO: support different types of failures/errors
+enum class WorkResult : std::uint8_t {
+  SUCCESS = 0,
+  FAILURE = 1,
+  UNKNOWN = 100,
```

Comment:
nit: why jump from 1 to 100?

Reply:
no special meaning, just followed the enum tradition the line of 34 (enum class OpType) for unknown result.